bar-trender-front-end/src/components/ModalSelectedElement.js

```diff
@@ -1,18 +1,64 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
+import {
+  Row,
+  Col,
+} from "reactstrap";
 
 // reactstrap components
 import {
   Button,
   Modal,
   ModalBody,
+  ModalFooter,
 } from "reactstrap";
 
 // core components
 import image_left from '../assets/img/bg-landing.png';
+import Qr from '../App.js'
+
+//css components
+import '../views/css/ModalQR.css'
+import ListDiscount from "./ListDiscount";
 
 function ModalSelectedElement(prop) {
   const [modal1, setModal1] = React.useState(false);
+  const [modal2, setModal2] = React.useState(false);
   const { element } = prop;
+  const [appState, setAppState] = useState({
+    discounts: {},
+  });
+
+   useEffect(() => {
+    const apiUrl = "https://develop-backend-sprint-01.herokuapp.com/v1/establishments/"+element.id+"/discounts/get";
+    async function loadDiscounts(){
+      await fetch(apiUrl, {
+        method: 'GET',
+        headers: {
+          'Content-Type': 'application/json',
+          'token': '',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7138516)
```suggestion
          "token": '',
```